src/Traits/PayPalRequest.php

```diff
@@ -120,13 +120,19 @@ private function setConfig(array $config = [])
      *
      * @return void
      */
-    protected function setClient()
+    protected function setClient($certificate = '')
```

Comment:
I also wouldn't pass a certificate parameter here, perhaps adding a variable `$apiCert` would be better.